packages/host/app/components/matrix/room.gts

```diff
@@ -239,6 +241,24 @@ export default class Room extends Component<Signature> {
     registerDestructor(this, () => {
       this.scrollState().messageVisibilityObserver.disconnect();
     });
+
+    this.setRoomLLM();
```

Comment:
This looks like it running the constructor of the room component. It seems like a surprising place to me that might miss certain cases. For example, creating a new room, or switching rooms?

Reply:
My reasoning behind doing it this way is because I thought this made sense only for newly created rooms. But now I see it would be better to support switching rooms too, and move the logic out of the constructor. This is now fixed in later commits